### ISSUE TITLE
Integration of MVIKotlin for sharing code using MVI pattern

### ISSUE THREAD

**Issue by SamuelMarks** [arkivanov/decompose-multiplatform-template#22]
Since cc49e79e1564b69805df52147dc0c6c1a899bc48 in August of last year you've had this on your TODO list.

Would be great to see! - Personally, I am interested in basic navigation, gathering asynchronous data—e.g., from camera or network—and populating screens with new data. Hacking through lots of sample apps and I've almost got everything working, just MviKotlin left to integrate.

Thanks for building great libraries

**Comment by arkivanov** [comment]
Yeah, the readme is a bit outdated. Take a look at the following sample: https://github.com/IlyaGulya/TodoAppDecomposeMviKotlin. Add the sample to the readme.
